recurly/resource.py

```diff
@@ -216,6 +216,36 @@ def __init__(self, **kwargs):
             if key not in ('collection_path', 'member_path', 'node_name', 'attributes'):
                 setattr(self, key, value)
 
+    def _beauty_print(self):
```

Comment:
Was `pretty_print` taken? 😅

Reply:
maybe ahahaha, this just prints like ruby does. now the print looks better.

![image](https://user-images.githubusercontent.com/7256891/184443626-89dd19ca-3ff2-4f78-a038-0f5db5e096f0.png)